CertPathValidatorException

My guess with this one is that someone is trying to connect to a site that has a self-signed or invalid SSL certificate:

```
org.wordpress.android.util.HttpRequest$HttpRequestException: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
at org.wordpress.android.util.HttpRequest.code(HttpRequest.java:1381)
at org.wordpress.android.util.HttpRequest.stream(HttpRequest.java:1638)
at org.wordpress.android.util.HttpRequest.buffer(HttpRequest.java:1627)
at org.xmlrpc.android.ApiHelper.getResponseStream(ApiHelper.java:857)
at org.wordpress.android.ui.accounts.AccountSetupActivity.getRSDMetaTagHref(AccountSetupActivity.java:598)
at org.wordpress.android.ui.accounts.AccountSetupActivity.configureAccount(AccountSetupActivity.java:181)
at org.wordpress.android.ui.accounts.AccountSetupActivity.access$1100(AccountSetupActivity.java:59)
at org.wordpress.android.ui.accounts.AccountSetupActivity$2.run(AccountSetupActivity.java:676)
Caused by: javax.net.ssl.SSLHandshakeException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
at org.apache.harmony.xnet.provider.jsse.OpenSSLSocketImpl.startHandshake(OpenSSLSocketImpl.java:413)
at org.apache.harmony.xnet.provider.jsse.OpenSSLSocketImpl.startHandshake(OpenSSLSocketImpl.java:257)
at libcore.net.http.HttpConnection.setupSecureSocket(HttpConnection.java:210)
at libcore.net.http.HttpsURLConnectionImpl$HttpsEngine.makeSslConnection(HttpsURLConnectionImpl.java:477)
at libcore.net.http.HttpsURLConnectionImpl$HttpsEngine.connect(HttpsURLConnectionImpl.java:432)
at libcore.net.http.HttpEngine.sendSocketRequest(HttpEngine.java:282)
at libcore.net.http.HttpEngine.sendRequest(HttpEngine.java:232)
at libcore.net.http.HttpURLConnectionImpl.getResponse(HttpURLConnectionImpl.java:273)
at libcore.net.http.HttpURLConnectionImpl.getResponseCode(HttpURLConnectionImpl.java:479)
at libcore.net.http.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:133)
at org.wordpress.android.util.HttpRequest.code(HttpRequest.java:1379)
... 7 more
Caused by: java.security.cert.CertificateException: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
at org.apache.harmony.xnet.provider.jsse.TrustManagerImpl.checkTrusted(TrustManagerImpl.java:184)
at org.apache.harmony.xnet.provider.jsse.TrustManagerImpl.checkServerTrusted(TrustManagerImpl.java:163)
at org.apache.harmony.xnet.provider.jsse.OpenSSLSocketImpl.verifyCertificateChain(OpenSSLSocketImpl.java:593)
at org.apache.harmony.xnet.provider.jsse.NativeCrypto.SSL_do_handshake(Native Method)
at org.apache.harmony.xnet.provider.jsse.OpenSSLSocketImpl.startHandshake(OpenSSLSocketImpl.java:410)
... 17 more
Caused by: java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.
... 22 more
```

--- a/src/org/xmlrpc/android/ApiHelper.java
+++ b/src/org/xmlrpc/android/ApiHelper.java
@@ -860,10 +860,13 @@ public class ApiHelper {
     public static InputStream getResponseStream(String urlString) {
         HttpRequest request = getHttpRequest(urlString);
         if (request != null) {
-            return request.buffer();
-        } else {
-            return null;
+            try {
+                return request.buffer();
+            } catch (HttpRequestException e) {
+                Log.e( "ApiHelper", "Cannot setup an InputStream on " + urlString, e );
+            }
         }
+        return null;
     }
 
     /**
@@ -879,6 +882,7 @@ public class ApiHelper {
                 String body = request.body();
                 return body;
             } catch (HttpRequestException e) {
+                Log.e( "ApiHelper", "Cannot load the content of " + urlString, e );
                 return null;
             }
         } else {

